src/views/infra/zone/ZoneWizardZoneDetailsStep.vue

```diff
@@ -120,6 +120,43 @@
           }]"
         />
       </a-form-item>
+      <a-form-item
+        :label="$t('ip6cidr')"
+        v-bind="formItemLayout"
+        v-if="isAdvancedZone && securityGroupsEnabled"
```

Comment:
Is this setting/field only taken into consideration when SG is enabled @GabrielBrascher ?

Reply:
Sorry for the late reply @rhtyd.

So far there is no support for IPv6 via VPC/VRs. Therefore, the zone must be an Advanced zone with Security Group enabled to handle IPv6 allocation and ACLs.

Thanks for the review!